diff -Nurp a/VisPyPatches.py b/VisPyPatches.py
--- a/appGUI/VisPyPatches.py	2024-11-13 16:57:45.000000000 +0100
+++ b/appGUI/VisPyPatches.py	2024-11-13 16:58:43.000000000 +0100
@@ -29,8 +29,8 @@ def apply_patches():
     }
     """
 
-    markers._marker_dict['++'] = cross_lines
-    markers.marker_types = tuple(sorted(list(markers._marker_dict.copy().keys())))
+    #markers._marker_dict['++'] = cross_lines
+    #markers.marker_types = tuple(sorted(list(markers._marker_dict.copy().keys())))
 
     # # Add clear_data method to LineVisual to have possibility of clearing data
     # def clear_data(self):
